config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -246,7 +246,7 @@ presubmits:
         - --provider=skeleton
         - --deployment=kind
         - --kind-binary-version=build
-        - --kind-config-path=./../../k8s.io/kubeadm/tests/e2e/kind/single-cp/single-cp.yaml
+        - --kind-config-path=./../../k8s.io/kubeadm/tests/e2e/kind/ha-cp/ha-cp.yaml
         - --build=bazel
```

Comment:
I'm not sure we should change kubernetes-security/generated-security-jobs.yaml jobs.
I don't know who owns them and what they are meant to
@neolit123 have you got some context here?

Reply:
the security jobs are auto-generated via ./hack/update-config.sh
if the script is not updating them we should not touch them.

@RA489 
is this change the result of the script?

Reply:
@neolit123 ./hack/update-config.sh script updated the security jobs.